charters/DocumentationCharter.md

```diff
@@ -0,0 +1,49 @@
+# Documentation Working Group Charter
+
+## Mission and Goals
+
+The Documentation Working Group ("Docs Working Group") serves as a support, helper, and advisory body to Jupyter Subprojects on all aspects of documentation. The core pillars of our mission:
+
+- Improve all aspects of documentation across the Jupyter ecosystem.
+- Make high quality documentation that is clear, comprehensive, inclusive, and serves the varying needs of Jupyter's diverse community.
+- Engage with the community to help users with Jupyter products, discover their needs, and connect them with relevant information, expertise and resources.
+
+The Docs Working Group will provide a place for consistent, focused, holistic efforts to be spent on documentation across the whole ecosystem. This group exists in part to provide capacity and resources to the subprojects (some of which are already suffering from a lack of resources/capacity, and more specifically to work on documentation in particular).
+
+We want all users to have positive experiences inside the Jupyter ecosystem, especially users who are learning and coming in for the first time, and users with disabilities.
+
+## Areas of Responsibility
+
+The Docs Working Group is responsible for the following:
+
+- Setting standards for documentation where needed.
+- The Docs working group will take ownership of documentation and related resources not owned by a Subproject (including the Jupyter.org website and documentation).
```

Comment:
When the people involved in this WG proposal joined the EC office hours, I believe we discussed that the WG would own all documentation not part of subprojects, including the documentation related parts of jupyter.org, but I don't believe we talked about the WG owning all of jupyter.org. I don't think that makes sense as the website has a ton of content that is unrelated to documentation (sponsorship, "marketing", messaging, fundraising, governance, etc.).

Reply:
It's clear to me that we need to make some further edits to the charter, and to clarify some points of confusion (my own and others) around some related issues.

Right now, the charter says (under "Areas of Responsibility"):

"The Docs working group will take ownership of documentation and related resources not owned by a Subproject (including the Jupyter.org website and documentation)."

This was based directly on @ellisonbg's feedback included here, below:

"One question that I have is that Jupyter has some documentation that isn't (currently) part of subprojects. Will this WG own that documentation? Right now this type of documentation is in the grey area of the project where it isn't clear who can even review/merged PRs. I think it would be helpful if the charter gave the WG ownership of those docs."

We can revise the charter to say "The Jupyter.org documentation sites (under the current 'Documentation' header item)" or similar, which should help narrow the scope and meaning of that section. Do we have consensus on this narrowed clause? I think it makes sense for the Docs Working Group to own that, and it should be close to the meaning of @ellisonbg's original suggestion.

Beyond that, it was originally my (seemingly wrong) impression that the larger Jupyter.org site was another mostly orphaned Jupyter asset (I'd heard in the weekly meetings that the site needed refactoring but that nobody was available to do it). So it seems my impression was wrong, and that the EC owns the Jupyter.org site and has existing procedures/resources around content and maintenance...is that accurate?

I'm comfortable removing the larger "Jupyter.org website" from our charter and sticking with the narrower "Jupyter.org Documentation Sites (under the current Documentation header)".

Reply:
Some additional thoughts about this thread and @zsailer’s comments above:

Jupyter (the project) is home to many key pieces that are critical to its success.

Is Server more or less important than Notebook? Or Lab, or Hub? That's a ridiculous question, akin to picking the heart over the brain, or kidneys over the lungs. They're all important, and add different things that are important to different stakeholders (contributing to the overall success and health of the project).

At the same time, none of these pieces is MORE important than the project as a whole. We give each Subproject a lot of independence and autonomy, which is part of what makes our collaborative open source work culture so successful (and each subproject so robust/vibrant). But when stakeholders from different Subprojects feel like they’re competing (for space on the home page for instance), it’s detrimental to the project as a whole.

If everyone is shouting at the top of their lungs, nobody is heard. The listener (or reader), instead of taking away a few limited but important pieces of information about Jupyter, is left with a jumbled incoherent mess of competing minutiae.

In my mind, this (fragmentation) is why we keep hearing the same confusion/feedback from users, described well by @rpwagner via security stakeholders in the original Docs Working Group proposal thread:

> …one of the repeated pieces of feedback we've heard from groups deploying Jupyter components (JupyterLab, JupyterHub, etc.) who aren't active in the community is that it's challenging to get a unified view of "what is Jupyter" and how the components work together.

This is why we need Jupyter-wide groups like the Docs Working Group, to unify and harmonize what has become a very fragmented, confusing experience for our users.

To that end, in my mind, people who come to Jupyter.org should understand that:

- The Jupyter project has many key pieces, all about interactive computing with computational notebooks
- You can customize, improve and extend Jupyter software to do just about anything you need

If we can’t agree on these or a similar simple, clear communication hierarchy of ideas about Jupyter to communicate (rather than having subprojects all competing to cram the universe into the first sentence), our users will remain lost and confused.

Reply:
> We can revise the charter to say The Jupyter.org documentation sites (under the current 'Documentation' header item) or similar, which should help narrow the scope and meaning of that section. Do we have consensus on this narrowed clause? 

Yep, this phrasing looks good to me. 

BTW, don't get me wrong. I'm not against the WG having more decision-making power on jupyter.org; I just think that's a bigger change than originally intended—so I wanted to flag it. 

> the EC owns the Jupyter.org site and has existing procedures/resources around content and maintenance...is that accurate?

That's my point above; I don't know if this is true. Here's what the governance docs say about the EC:

![Screenshot 2024-01-04 at 2 12 23 PM](https://github.com/jupyter/governance/assets/2791223/8b1a513b-0781-4c11-bcbd-d1d302cbc348)

The EC "manages" the jupyter.org service; but I interpret that to mean they ensure that the domain is always available and hosted appropriately. They don't have sole ownership over what content gets posted there. 

Up until this point, content was always been decided by "consensus", but as the site has grown, _consensus_ has been more difficult to obtain. 

> I'd heard in the weekly meetings that the site needed refactoring but that nobody was available to do it

I don't think it's an issue of availability; rather, it's the consensus problem. 

> If we can’t agree on these or a similar simple, clear communication hierarchy of ideas about Jupyter to communicate (rather than having subprojects all competing to cram the universe into the first sentence), our users will remain lost and confused.

I think most (if not everyone) agrees with this sentiment, but it's difficult in practice.  There are many different answers to the question "what is Jupyter?". In fact, there is a whole class of "Jupyter users" who have never touched a computational Notebook. They use kernels and nothing else—in fact, Jupyter started as an interactive shell/console :wink: 

Interactive computing is definitely a core theme; but that's likely not concrete enough. 

That said, definitely take a look at the JEP for a re-design; I think it attempts to capture the docs experience you're describing and you'll appreciate. I'd love to see someone pick that effort up again and push it through. Even if we didn't want to use the design assets proposed there, we could learn a lot from the logical hierarchy of the site they proposed. 

>  But when stakeholders from different Subprojects feel like they’re competing (for space on the home page for instance), it’s detrimental to the project as a whole.

I think there's subtle difference from the picture you describe here and what I've seen happen.

It's often not the _developers_ of the subprojects demanding a higher position on the "Jupyter org" chart; it's our attempt to document (and failure to agree on) what users look to Jupyter to provide. 

Our user community is diverse when answering the question "What is Jupyter?" or "what does Jupyter mean to you?". e.g. some think Lab, some like Notebook. Some think IPython shell, some think interactive webpages (Voila/Lite/etc). Some think Cloud, some think localhost. Some think web app, others think desktop.

Reply:
@zsailer The redesign JEP seems to pretty successfully communicate the diverse offerings and use cases of Jupyter products :) And the Docs WG can definitely take the lead on the Jupyter.org redesign/JEP. It's a natural fit because it's an important resource that helps the outside world understand the Jupyter ecosystem (which is tied to the Docs WG's mission).

I've observed the difficulties obtaining consensus as I've onboarded to Jupyter work these past ~2 years: We're often left stalled on important work, or as is often the case with docs- or communication- related efforts, we end up with outputs that don't communicate effectively because too much has been included, and what's there has not been structured according to a coherent underlying vision of what concepts are most important (and I hope to develop some of these in collaboration with all the subprojects).

Jupyter-wide challenges like the consensus problem are likely to grow over time in step with the project's growth. I'm hoping the Docs WG can help innovate solutions for working through these type of organization-wide problems (maybe there are some reusable work processes to discover and share here).

These issues, in my mind, make a clearer case for the creation of new Jupyter-wide bodies (like the DWG) that act to add consistency, unity and cohesion to Jupyter offerings (and to counteract any fragmentation that arises between subprojects).

By ensuring these new groups are highly collaborative and deeply connected to the subprojects, and by acting from a project-wide perspective that takes the health of the overall project as a priority, my hope is that we can more effectively solve these sorts of systemic problems.

Reply:
> These issues, in my mind, make a clearer case for the creation of new Jupyter-wide bodies (like the DWG) that act to add consistency, unity and cohesion to Jupyter offerings (and to counteract any fragmentation that arises between subprojects).

💯 🚀  I love it, @ericsnekbytes! Thank you for taking this on—you've got my support!